app/javascript/controllers/navbar_controller.js

```diff
@@ -0,0 +1,6 @@
+const navToggle = document.getElementById('nav-toggle');
+const mobileMenu = document.getElementById('mobile-menu');
+
+navToggle.addEventListener('click', () => {
+    mobileMenu.classList.toggle('hidden');
+});
```

Comment:
I would generally recommend keeping with Stimulus here to keep things standardized, but it's hard to argue against the simplicity of this.

One downside to this approach is that it's a bit harder to see what's happening from the HTML. It looks as though this is binding an event handler to an image tag? Ideally this should at least be a button element to avoid accessibility issues.

Reply:
I agree with @JoshAntBrown, but agreed that it's super simple, so I like it. 🙂 It's up for grabs if someone wants to refactor in the futre.